Content.Server/Station/StationSystem.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Maps;
+using Content.Shared.Station;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+
+namespace Content.Server.Station;
+
+/// <summary>
+/// System that manages the jobs available on a station, and maybe other things later.
+/// </summary>
+public class StationSystem : EntitySystem
+{
+    [Dependency] private GameTicker _gameTicker = default!;
+    private uint _idCounter = 1;
+
+    private Dictionary<StationId, StationInfoData> _stationInfo = new();
+    /// <summary>
+    /// List of stations for the current round.
+    /// </summary>
+    public IReadOnlyDictionary<StationId, StationInfoData> StationInfo => _stationInfo;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<GameRunLevelChangedEvent>(OnRoundEnd);
+    }
+
+    /// <summary>
+    /// Cleans up station info.
+    /// </summary>
+    private void OnRoundEnd(GameRunLevelChangedEvent eventArgs)
+    {
+        if (eventArgs.New == GameRunLevel.PostRound)
+            _stationInfo = new();
+    }
```

Comment:
if you clear _stationInfo, you might as well reset _idCounter, or is there some reason to distinguish the stations in the new round?

Reply:
Just in case something accidentally reuses the ID. Same reason we don't reuse entity IDs.

Reply:
fair enough